pkg/ibm/resources_container.go

```diff
@@ -60,20 +61,20 @@ func (tagsCollector *tagsClient) setResourceTags(resource datamodel.TaggedResour
 // ResourcesContainer holds the results of collecting the configurations of all resources.
 type ResourcesContainer struct {
 	datamodel.ResourcesContainerModel
-	regions         []string
-	ResourceGroupID string
+	regions               []string
+	ResourceGroupIDOrName string
```

Comment:
```suggestion
	resourceGroupIDOrName string
```
No need to export this field